packages/x-date-pickers/src/internals/utils/text-field-helper.ts

```diff
@@ -79,9 +79,33 @@ export function checkMaskIsValidForCurrentFormat(
     inferredFormatPatternWith2Digits === mask && inferredFormatPatternWith1Digits === mask;
 
   if (!isMaskValid && utils.lib !== 'luxon' && process.env.NODE_ENV !== 'production') {
-    console.warn(
-      `The mask "${mask}" you passed is not valid for the format used ${format}. Falling down to uncontrolled not-masked input.`,
-    );
+    const defaultWarning = [
+      `The mask "${mask}" you passed is not valid for the format used ${format}.`,
+      `Falling down to uncontrolled no-mask input.`,
+    ];
+
+    if (format.includes('MMM')) {
```

Comment:
Not sure to understand why we do this `includes('MMM')` check

Reply:
OK, is this incompatibility impossible to change (in the future) or is it just a quick fix waiting for a rework ?

We are in the `if(utils.lib == 'luxon')` is it intended ?

Reply:
It's not impossible. For later we could imagine that typing `'1 j'` proposes to complete with `'1 jun'` `'1 jul'` or `'1 jan'` but not for now.

Seems that luxon library was causing problems. I don't know why they decided to do hide the warning message for this specific library